templates/ntp.conf.erb

```diff
@@ -32,7 +32,7 @@ filegen clockstats file clockstats type day enable
 # pool: <http://www.pool.ntp.org/join.html>
 
 <% @servers.each do |server| -%>
-server <%= server %><% if @server_options != 'UNSET' %> <%= @server_options %><% end %>
+server <%= server %><% if @server_options != 'UNSET' %> <% @server_options.each do |option| -%><%= option %><% end -%><% end %>
```

Comment:
This is separate from adding the interface parameter and should be in its own pull request.

I noticed you are using your master branch instead of creating a topic branch. Suggest creating a topic branch so this can be separate from the interface pull request (PR).